gadgets/src/integers/uint/arithmetic/sub.rs

```diff
@@ -53,71 +46,6 @@ macro_rules! sub_int_impl {
 
                 Self::addmany(&mut cs.ns(|| "add_not"), &[self.clone(), other.negate()]).map_err(|e| e.into())
             }
-
```

Comment:
Explain: Since the `sub_unsafe` function cannot work in the setup mode, the new Div gadget no longer uses it. Therefore, this function is dropped.